tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "declaration": true,
     "outDir": "./dist",
     "strict": true,
-    "noImplicitAny": false,
+    "noImplicitAny": true,

```

Comment:
I feel like having implicit `any`s defeats the point of TS

Reply:
No, just makes types optional. I just made this `false` when I was prototyping but probably a good idea to make it true now